test-case/multiple-pipeline-capture.sh

```diff
@@ -147,7 +149,7 @@ do
     [[ $? -eq 1 ]] && func_error_exit "Catch the abnormal process status of aplay"
 
 
-    # kill all arecord
+    dlogi 'pkill -9 aplay arecord'
```

Comment:
`dlogc` here maybe? dlogi should be some message, but this looks like more like exactly cmd.